### ISSUE TITLE
License for repo

### ISSUE THREAD

**Issue by mchelen** [okfn/okfn.github.com#256]
What is the license for this repo, MIT?

**Comment by rgrp** [comment]
@mchelen do you want to apply the MIT to the repo and make clear that applies to code.

**Comment by rgrp** [comment]
@mchelen are you happy to action this one?

**Comment by tfmorris** [comment]
As a matter of policy and best practice, it would be good to have license declared at the time of repo creation for OKFN reps.  That way contributors can make sure that they're not adding code with conflicting license terms (and that the license is something that they're happy to contribute under).

**Comment by rgrp** [comment]
@tfmorris def agree in general - point here was lack of code / ref to main terms of use.

**Comment by mchelen** [comment]
Would it ok to just include a copy of CC-BY? I'm only seeing content (not code) in this repo.

**Comment by rgrp** [comment]
@mchelen yes - or even just reference via URL at latest version. I also recommend linking to https://okfn.org/terms-of-use/ and http://okfn.org/ip-policy as we do in footer of the site.
